securitycenter/api/src/UpdateNotificationConfigSnippets.g.cs

```diff
@@ -33,10 +33,11 @@ public class UpdateNotificationConfigSnippets
         {
             NotificationConfigName = notificationConfigName,
             Description = "updated description",
-            PubsubTopicAsTopicName = pubsubTopic
+            PubsubTopicAsTopicName = pubsubTopic,
+            StreamingConfig = new StreamingConfig { Filter = "state = \"INACTIVE\"" }
```

Comment:
This now won't build because the compiler doesn't know where to find StreamingConfig. (That's why the tests are failing).
You need to add a using clause at the top of the samples, something like:
`using static NotificationConfig.Types;` The development environment that you are using should know which using you need to add.

Reply:
Still not building:
```
UpdateNotificationConfigSnippets.g.cs(21,14): error CS0246: The type or namespace name 'NotificationConfig' could not be found (are you missing a using directive or an assembly reference?) [/tmpfs/src/github/dotnet-docs-samples/securitycenter/api/src/Snippets.csproj]
    0 Warning(s)
    1 Error(s)
```

You need to add the correct using, it's going to be similar to `using static NotificationConfig.Types`  but right now there's something missing. Please do make sure that the code builds before pushing again, you can use your code editor for that (VisualStudio, VS Code, etc.) or simply run `dotnet build` on the command line on the project folder.